### DISCUSSION TITLE
Add support for data links in table headers

### DISCUSSION THREAD

**Discussion by dfernandez-dev** [grafana/grafana#37856] [Dashboards, Panels & Visualizations]
At the moment, tables support data links to other dashboards.
When set, all the values of the cells of the selected column/s are displayed as links.
It would be nice if header names also support data links. Or maybe give the user the option to add links to cells or to headers.

Although it seems like a "simple" feature, the table headers do sorting on click so it might be tricky to have both working at the same time.

Image attached as reference. I'd like to be able to add a link to each location name instead of to their values:

![image](https://user-images.githubusercontent.com/8043341/129278730-e91e240c-c05c-430f-85c3-7b8efeebf840.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
